### ISSUE TITLE
Exception encountered when loading plugin: NuVotifier

### ISSUE THREAD

**Issue by andrewkm** [parallelblock/NuVotifier#93]
Just updated nuvotifier. Last update was May 27th from:
https://repo.parallelblock.com/repository/public-raw/nuvotifier/nuvotifier.jar
I'm now getting this:

Reverting fixes my issue.
Did something change?

```
05:28:09 [INFO] [NuVotifier] Loaded token for website: default
05:28:09 [INFO] [NuVotifier] Votifier enabled on socket /0:0:0:0:0:0:0:0:8192.
05:28:09 [WARNING] Exception encountered when loading plugin: NuVotifier
java.lang.NoSuchMethodError: com.google.gson.JsonObject.keySet()Ljava/util/Set;
	at com.vexsoftware.votifier.bungee.forwarding.cache.FileVoteCache.load(FileVoteCache.java:70)
	at com.vexsoftware.votifier.bungee.forwarding.cache.FileVoteCache.<init>(FileVoteCache.java:32)
	at com.vexsoftware.votifier.bungee.NuVotifier.loadAndBind(NuVotifier.java:258)
	at com.vexsoftware.votifier.bungee.NuVotifier.onEnable(NuVotifier.java:321)
	at net.md_5.bungee.api.plugin.PluginManager.enablePlugins(PluginManager.java:227)
	at net.md_5.bungee.BungeeCord.start(BungeeCord.java:281)
	at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:56)
	at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)
```

**Comment by astei** [comment]
I noticed this issue too and fixed it in #94. If/when that's merged in this will be fixed too.

**Comment by andrewkm** [comment]
Awesome! Thanks a lot @astei
